client/src/main.rs

```diff
@@ -790,6 +790,50 @@ fn main() {
                     println!("{:?}", offerings);
                     Ok(())
                 }),
+        ).add_cmd(
+            Command::new("endorse-newcomer")
+                .description("endorses a newcomer")
+                .options(|app| {
+                    app.setting(AppSettings::ColoredHelp)
+                    .account_arg()
```

Comment:
account will be the sender of the extrinsic. you need to an additional argument, which will be the endorsee account